### PR TITLE
[14.0][FIX] pos_cash_move_reason: do not load account.account from demo data

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Loading account creation from demo data could have side effect when a chart of account is loaded afterwards.
See: https://github.com/odoo/odoo/blob/14.0/addons/account/models/chart_template.py#L205

### CODE REVIEW

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py [hunk 1]
```diff
@@ -1,14 +1,30 @@
 # © 2015 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+from odoo import tools
 from odoo.exceptions import UserError
+from odoo.modules.module import get_module_resource
 from odoo.tests.common import Form, SavepointCase
 
 
+def load_file(cr, module, *args):
```
Comment (legalsylvain) [inline pos_cash_move_reason/tests/test_pos_cash_move_reason.py:10]: There is no core function that do that ?
Just a question...
Otherwise lgtm.

**Review by legalsylvain — APPROVED** [review]

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py (hunk 1 above)
Comment (Cedric-Pigeon) [reply pos_cash_move_reason/tests/test_pos_cash_move_reason.py:10]: I do not think so

**Review by ivantodorovich — APPROVED** [review]

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Actually, can you check?
I think the `demo` files are still there

**Comment by ivantodorovich** [comment]
@Cedric-Pigeon could you remove these files, as they should've been moved instead:

https://github.com/acsone/pos/tree/14.0-pos_reason-cpi/pos_cash_move_reason/demo

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py (hunk 1 above)
Comment (rousseldenis) [reply pos_cash_move_reason/tests/test_pos_cash_move_reason.py:10]: Maybe we can use this : https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L255

But not the same 'context' as we are in tests here

**Comment by elvise** [comment]
> @Cedric-Pigeon could you remove these files, as they should've been moved instead:
> 
> https://github.com/acsone/pos/tree/14.0-pos_reason-cpi/pos_cash_move_reason/demo

@Cedric-Pigeon can you check this point ?

**Pull request closed without merging**